#Information about the operation status from the local planner

Header header
Bool has_reached_goal
Bool is_stuck
Bool cannot_plan_path
